.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,33 @@
+name: Process Dependabot PRs
+
+permissions: write-all
+
+on: [pull_request]
+
+jobs:
+
+  process_dependabot_pr:
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Run version bump script
+        run: |
+          bash dependency-version-bump.sh
+
+      - name: Commit changes
+        run: |
+          git config --local user.email "action@github.com"
```

Comment:
I don't think we should reuse dependabots credentials if we are changing the commit.